File: AirsofterAPI/AirsofterAPI/DTO/UserDTO.cs
namespace AirsofterAPI.DTO
{
    public class UserDTO
    {
        public int Id { get; set; }
        public string? Username { get; set; }
        public string? DisplayName { get; set; }
        public string? Password { get; set; }
        public string? Email { get; set; }
        public DateTime CreationDate { get; set; }

    }
}

File: AirsofterMobile/app/src/main/java/com/airsofter/airsoftermobile/login/data/network/model/UserDTO.kt
package com.airsofter.airsoftermobile.login.data.network.model

import com.google.gson.annotations.SerializedName

data class UserDTO(
    @SerializedName("id") val id: Int,
    @SerializedName("username") val username: String,
    @SerializedName("displayName") val displayName: String,
    @SerializedName("email") val email: String
)
